src/test/codewhisperer/util/crossFileContextUtil.test.ts

```diff
@@ -169,7 +169,10 @@ describe('crossFileContextUtil', function () {
 
                 const actual = await crossFile.fetchSupplementalContextForSrc(editor, fakeCancellationToken)
 
-                assert.ok(actual?.length !== undefined && actual.length === 0)
+                assert.ok(
+                    actual?.supplementalContextItems.length !== undefined &&
+                        actual.supplementalContextItems.length === 0
+                )
```

Comment:
nit, I think this works and is equivalent?

```suggestion
                assert.ok(actual?.supplementalContextItems.length === 0)
```

or if that doesn't work,

```suggestion
                assert.ok(actual !== undefined && actual.supplementalContextItems.length === 0)
```

Reply:
yea make sense